civihr_employee_portal/views/templates/_views-view--generic-display-output.tpl.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
@powdevel file name doesn't look good to me why its starting with _ ?

Reply:
@hitesh-jain this a common template which its name does not come from a template suggestion. The `_` was deliberate to point that this template is different, something like a helper for other templates. This template is called from the default views template only for certain views, and was built to mimic a panel block template, to integrate views transparently on panel pages, to re-use current styles for pages like the ones used on dashboard.
PD: I'm adding this comment on the technical description and this template file.